packages/datadog_flutter_plugin/lib/datadog_flutter_plugin.dart

```diff
@@ -265,8 +247,8 @@ class DatadogSdk {
     });
   }
 
-  /// Determine if the provided URI is a first party host as determined by the
-  /// value of [firstPartyHosts].
```

Comment:
In Swift `///` allows introspection of a comment if you option click the property.

Not sure if it works this way in dart.

Reply:
Yes triple slash is the Dart Doc indicator.